server/ai_process.go

```diff
@@ -864,6 +865,32 @@ func submitAudioToText(ctx context.Context, params aiRequestParams, sess *AISess
 	return &res, nil
 }
 
+func submitLiveVideoToVideo(ctx context.Context, params aiRequestParams, sess *AISession, req worker.StartLiveVideoToVideoFormdataRequestBody) (*worker.StartLiveVideoToVideoResponse, error) {
+	client, err := worker.NewClientWithResponses(sess.Transcoder(), worker.WithHTTPClient(httpClient))
+	if err != nil {
+		if monitor.Enabled {
+			monitor.AIRequestError(err.Error(), "LiveVideoToVideo", *req.ModelId, sess.OrchestratorInfo)
+		}
+		return nil, err
+	}
+	resp, err := client.StartLiveVideoToVideoWithFormdataBodyWithResponse(ctx, req)
+	if err != nil {
+		return nil, err
+	}
+	if resp.JSON200 != nil {
+		fmt.Println("publish", resp.JSON200.PublishUrl, "subscribe", resp.JSON200.SubscribeUrl)
+	}
+	if resp.JSON400 != nil {
+		fmt.Println("json400", resp.JSON400)
+	}
+	if resp.JSON500 != nil {
+		fmt.Println("json500", resp.JSON500)
+	}
+	fmt.Println("body is", string(resp.Body))
```

Comment:
If you plan to merge, then please change to `clog` or remove.

Reply:
Removed in https://github.com/livepeer/go-livepeer/commit/9e461a7d2ee59b1cd5603382d403c082b71469ad